### PR TITLE
Release notes for the prod deployment on May 8th, 2019

### PR DESCRIPTION
**Pull request by rhiananthony**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,6 +2,128 @@
...
+* Sync daemon sets destination content-type equal to source
+* Update subscription_v2 ddb interface calls
+* Use dynamoDB methods in async_state
+* Abstract out dynamoDB functionality from subscriptions
+
+## Secondary Analysis
+### Pipeline-tools v0.49.1
+* Update metadata schema when formatting analysis results:
+* Change process/protocol_type to type
+* Change file_format to format
```
Comment (rexwangcc) [inline RELEASE_NOTES.md]: The above 2 lines need to be indented.

Comment (rhiananthony) [reply RELEASE_NOTES.md]: Done

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -2,6 +2,128 @@
...
+    * Change file_format to format
+* Update to metadata-api library release/1.0b15
+* Subscriptions queries process_type -> type, dissociation_method -> method
+
+## Data Browser
+### Version: 156f751de3b693ffa8dabcb78ee7550bcee5fe7f
+* Samples Tab
+
+## Azul
+### Version: 53f0b12c25364a8c5f527ad2119fee48df096b40
```
Comment (hannes-ucsc) [inline RELEASE_NOTES.md]:
```suggestion
### Version: 6a3639a97acc46cde7d98530c604f777ecee99fd
```

Sorry, for not filling out the release notes earlier. The individual commits are the same but the merge commit has a different sha.

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -2,6 +2,128 @@
...
+* Setting of row height automatically to show all validation errors
+* Do not allow creation of project metadata thru UI
+ 
+## Upload  v4.3.1
+* Remove "list upload area" REST API endpoint
+* Added client-side checksumming (temporarily disabled).
+* Much Internal refactoring and cleanup.
+
+## Data Store
+### Version: 2019-05-08-02-45-24-staging.release
```
Comment (xbrianh) [inline RELEASE_NOTES.md]: Version should be `2019-05-08-15-27-25-prod.release`

Comment (rhiananthony) [reply RELEASE_NOTES.md]: Thanks Brian! I changed it.

**Pull request merged**